editor/icons/NodeInfo.svg

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This needs to be [optimized](https://docs.godotengine.org/pl/stable/development/editor/creating_icons.html#icon-optimization).